bazel/foreign_cc/luajit.patch

```diff
@@ -73,6 +73,13 @@ index 0000000..9c71271
 +    # fail on it.
 +    os.environ["LSAN_OPTIONS"] = "exitcode=0"
 +
++    # Blacklist LuaJIT from ASAN for now.
++    # TODO(htuch): Remove this when https://github.com/envoyproxy/envoy/issues/6084 is resolved.
++    if "ENVOY_CONFIG_ASAN" in os.environ:
++      os.environ["TARGET_CFLAGS"] += " -fsanitize-blacklist=%s/com_github_luajit_luajit/clang-asan-blacklist.txt" % os.environ["PWD"]
```

Comment:
if we do `fun:*` blacklist, shall we just remove -fsanitize=address from TARGET_CFLAGS?

Reply:
It's a useful hook to leave around to experiment with making this more precise, so I'd vote to keep it as is.

Reply:
SGTM